### PR TITLE
Fix the output path when building a single test.

### PR DESCRIPTION
**Pull request by pgavlin**
Add a trailing slash to the base output path.

### CODE REVIEW

**Comment by pgavlin** [comment]
@ellismg @dagood PTAL

**Comment by ellismg** [comment]
LGTM

File: tests/src/dir.common.props [hunk 1]
```diff
@@ -40,7 +40,7 @@
     <BaseOutputPath Condition="'$(__TestRootDir)' != ''">$(__TestRootDir)\</BaseOutputPath>
     <BaseOutputPathWithConfig>$(BaseOutputPath)\$(OSPlatformConfig)\</BaseOutputPathWithConfig>
     <BinDir>$(BaseOutputPathWithConfig)</BinDir>
-    <BaseIntermediateOutputPath>$(ProjectDir)\..\bin\tests\obj\$(OSPlatformConfig)\Managed</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath>$(ProjectDir)\..\bin\tests\obj\$(OSPlatformConfig)\Managed\</BaseIntermediateOutputPath>
```
Comment (dagood) [inline tests/src/dir.common.props]: It looks like the convention used to be no trailing slashes, and now they're included, so maybe `\`'s after `$(ProjectDir)` and such should be removed? They aren't inserted after `$(ProjectDir)` in tests/dir.props for example.

**Comment by dagood** [comment]
LGTM, just a comment about other potential changes for consistency.

**Comment by pgavlin** [comment]
@mmitche: looks like the ARM emulator got stuck running tests: http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/arm_emulator_cross_release_ubuntu_prtest/2122/console

**Pull request merged**